frontend/package.json

```diff
@@ -5,7 +5,7 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "build-prod": "ng build --base-href /browser/",
```

Comment:
why did you remove the /browser base-href? I think this is needed when running the nginx container (see Dockerfile)

Reply:
I didn't know that the nginx container requires it. I reverted it.